app/src/docs/v1.api-spec.yaml

```diff
@@ -3830,6 +4284,57 @@ components:
             encryptionKey:
               $ref: '#/components/schemas/EncryptionKey'
         - $ref: '#/components/schemas/TimeStampUserData'
+    ExternalApi:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: ID of ExternalApi.
+          example: 1223h78b-21d2-4128-b4e8-02ad3daipoi32
+        formId:
+          type: string
+          format: uuid
+          description: Form ID this ExternalApi belongs to.
+          example: 1223h78b-21d2-4128-b4e8-02ad3daipoi32
+        name:
+          type: string
+          description: Name of the ExternalApi - unique per form.
+          example: BC Registries Number Lookup
+        endpointUrl:
+          type: string
+          format: url
+          description: Name of the ExternalApi - unique per form.
```

Comment:
Maybe change to "URL endpoint used for the external api"?

Reply:
See above. Thanks.